azure-cs-credential-rotation-one-set/Azure.CredentialRotation.OneSet.csproj

```diff
@@ -0,0 +1,19 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>netcoreapp3.1</TargetFramework>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Pulumi.Azure" Version="3.*" />
+    <PackageReference Include="Pulumi.AzureNextGen" Version="0.4.0" />
```

Comment:
Consider setting it to `0.*` or `0.6.0`.

Reply:
Fixed